src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -1916,6 +1928,12 @@ public Conversion ClassifyImplicitExtensionMethodThisArgConversion(BoundExpressi
                 {
                     return Conversion.ImplicitReference;
                 }
+
+                // PROTOTYPE: Investigate callers where Compilation is null.
+                if (Compilation is not null && HasImplicitSpanConversion(sourceType, destination, ref useSiteInfo))
```

Comment:
>Compilation [](http://example.com/codeflow?start=20&length=11)

`Compilation is null` from the `TypeConversions` derived type, which is used in `ReducedExtensionMethodSymbol.Create()` and other callers.

Reply:
Yes, I know, but I don't fully understand why TypeConversions are used there and whether it's fine that ImplicitSpan conversion won't be considered there.